### PR TITLE
Update Player download links for version 0.4.1

### PR DESCRIPTION
**Pull request by Ghabry**
Needs files first :D

### CODE REVIEW

**Comment by Ghabry** [comment]
Jenkins: Test this please

**Comment by carstene1ns** [comment]
What about moving the GCW Zero stuff to some historical section?

**Comment by Ghabry** [comment]
I was so kind to recompile it for 0.4.1 ;)

File: content/player/downloads.html [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
+	ver="0.4.1"
+	oldver="0.4.0"
 %>
 <div class="info">
 	<h2>Download EasyRPG Player</h2>
-	<h3>EasyRPG Player version <%= ver %> "Poison Attack"</h3>
-	<p>Release date: 2015-12-23.
-		<a href="/blog/2015/12/easyrpg-player-0-4-0-poison-attack/">
+	<h3>EasyRPG Player version <%= ver %> "Blind Attack"</h3>
+	<p>Release date: 2016-03-18.
```
Comment (carstene1ns) [inline content/player/downloads.html]: 2016-03-1**9**?

Comment (Ghabry) [inline content/player/downloads.html]: UTC+0 :D

**Comment by carstene1ns** [comment]
Links are ready now. Merging.

**Pull request merged**